core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1767,12 +1769,8 @@ protected final void closeNoLock(String reason, CountDownLatch closedLatch) {
         }
     }
 
-    private Releasable acquireLock(BytesRef uid) {
-        return keyedLock.acquire(uid);
-    }
-
     private Releasable acquireLock(Term uid) {
```

Comment:
I think we can clean this method up - it doesn't add much but craft and at least in one place we  go and do the same conversion later on - might as well capture it as a value on the caller

```
 try (Releasable ignore = acquireLock(get.uid())) { // we need to lock here to access the version map to do this truly in RT
 +                    versionValue = getVersionFromMap(get.uid().bytes());
 +                }
```

Reply:
I am not sure I follow